.travis.yml

```diff
@@ -1,20 +1,12 @@
-dist: trusty
-sudo: false
+dist: focal
 language: java
-jdk:
-- oraclejdk8
-install: "./gradlew --version"
-script: ./gradlew --no-daemon -DGRADLE_OPTS="-Xms128m" --stacktrace build -i
-after_success:
-- git status
-- echo $REPOSITORY_USERNAME
-- echo $TRAVIS_PULL_REQUEST
-- echo $TRAVIS_BRANCH
-- if [ master == $TRAVIS_BRANCH -a $TRAVIS_PULL_REQUEST == false -a  ${REPOSITORY_USERNAME:=X}
-  != X ]; then ./gradlew -DGRADLE_OPTS="-Xms128m" release -i; fi
+before_install:
+  - curl -L "http://nusmv.fbk.eu/distrib/NuSMV-2.6.0-zchaff-linux64.tar.gz" | tar --extract --gzip --strip-components=2 -C $HOME/bin "NuSMV-2.6.0-Linux/bin/NuSMV" || true
+  - curl -N -L "http://es-static.fbk.eu/tools/nuxmv/downloads/nuXmv-2.0.0-linux64.tar.gz" | tar --extract --gzip --strip-components=2 -C $HOME/bin "nuXmv-2.0.0-Linux/bin/nuXmv" || true
+install: 
+  - "gradle --version"
+script: travis_wait gradle --no-daemon -DGRADLE_OPTS="-Xms128m" --stacktrace build -i
```

Comment:
a couple of issues here; the SMV backend of Pardinus requires glibc 2.29 which in Travis is only available on more recent Ubuntu distributions, which unfortunately no longer support JDK8, so I had to use a more recent version of gradle.

also, the SMV backend relies of SMV model checkers that cannot be packaged with the Analyzer, so they are downloaded before building.